apparmor.d/groups/lxqt/lxqt-session

```diff
@@ -0,0 +1,115 @@
+# apparmor.d - Full set of apparmor profiles
+# Copyright (C) 2024 Alexandre Pujol <alexandre@pujol.io>
+# Copyright (C) 2024 Besanon  <m231009ts@mailfence.com>
+# SPDX-License-Identifier: GPL-2.0-only
+
+abi <abi/4.0>,
+
+include <tunables/global>
+
+@{exec_path} = @{bin}/lxqt-session
+profile lxqt-session @{exec_path} flags=(attach_disconnected) {
+  include <abstractions/base>
+  include <abstractions/dbus-accessibility>
+  include <abstractions/lxqt>
+  include <abstractions/qt5-shader-cache>
+  include <abstractions/nameservice-strict>
+
+  signal (send),
+  signal (receive) set=(kill, term) peer=startlxqt,
+  signal (receive) set=(kill, term) peer=sddm,
+
+  ptrace (read),
+
+  network netlink raw,
+
+  @{exec_path} mr,
+
+  @{sh_path}                          rix,
+  @{bin}/sed                          rix,
+  @{bin}/readlink                     rix,
+  @{bin}/dirname                      rix,
+  @{bin}/system-config-printer-applet rPx,
+  @{bin}/lxqt-config-input            rPx,
+  @{bin}/lxqt-session-settings        rPx,
+  @{bin}/lxqt-globalkeysd             rPx,
+  @{bin}/lxqt-panel                   rPx,
+  @{bin}/lxqt-policykit-agent         rPx,
+  @{bin}/lxqt-runner                  rPx,
+  @{bin}/lxqt-notificationd           rPx,
+  @{bin}/lxqt-powermanagement         rPx,
+  @{bin}/lxqt-config                  rPx,
+  @{bin}/lxqt-leave                   rPx,
+  @{bin}/lxqt-about                   rPx,
+  @{bin}/lxqt-config-monitor          rPx,
+  @{bin}/dbus-update-activation-environment rCx -> dbus,
+  @{bin}/systemctl                    rCx -> systemctl,
+
+  @{bin}/pavucontrol                  rPx,
+  @{lib}/geoclue-2.0/demos/agent      rPx,
+  @{bin}/python3.@{int}               rPx,
+  @{lib}/python3.@{int}               rPx,
```

Comment:
- There is no profile for `@{bin}/python3.@{int}`
- There is no such thing as an exec name: `@{lib}/python3.@{int}`

Reply:
for now i will remove this, yes. I used a different profile for python. Only applicable with some backup programs